### DISCUSSION TITLE
Are community Mondays still a thing?

### DISCUSSION THREAD

**Discussion by MartinGC94** [PowerShell/PowerShell#21138] [General]
Last year the team made an effort to bring down the number of open PRs by setting Mondays aside for reviewing PRs. For a while it seemed to work great and we went from about 100 open PRs to about 50 but at some point it stopped and the number of open PRs slowly grew to what we have today: 116.  
I assumed it was because the team was preparing for the LTS release but it got released mid November and now we are near the end of January with seemingly no change in pace (I only see 5 community PRs merged this month).

**Comment by kilasuit** [comment]
I expect this to continue to happen going forward however December/January are always a little slower especially more so after big releases like the LTS. This I have always guessed was down to the team not only needing to look through issues and PR's here but also things that have been raised via other means to them too, as well as start preparing their plans for the rest of the year. 

So I'd expect pace to pick back up in the next few weeks.

**Comment by kilasuit** [comment]
@MartinGC94 this was addressed in the latest community call.

**Reply by MartinGC94** [reply]
Thanks, link with timestamp for anyone interested: https://youtu.be/OJlJJnCtFrQ?t=876
The short version is that one of their goals for 7.5 is to get existing community PRs merged.  
Community Mondays are still a thing but there can be several reasons why we haven't seen as much action such as: Holidays, people taking personal time off, people being assigned work in other repos/projects, etc.

I won't dwell on the past so I'll just say that this sounds good and hopefully we will soon see some progress in this area.

**Comment by MartinGC94** [comment]
This has been closed as resolved. While it's definitely true that the initial question has been answered, the answer doesn't seem to be correct. Nothing's changed in the 2½ months since the question was answered.  
We are currently sitting at 123 open PRs. This Monday 1 community PR got merged, last Monday it was 0 and the Monday before that it was 1.

So the real answer is that community Mondays are effectively not a thing anymore.
